core/src/main/scala/io/snappydata/Literals.scala

```diff
@@ -144,35 +151,65 @@ object Property extends Enumeration {
       if (altName == null) name else name + '/' + altName
   }
 
-  type Type = ValueAlt
+  case class SQLValue(name: String, configEntry: SQLConfigEntry)
```

Comment:
Unapply is used in ServiceUtils. In fact it will be better to use extractors instead of multiple prop.startsWith checks and I will be adding better extractors.